drools-lsp-server/src/test/java/org/drools/lsp/server/LauncherTest.java

```diff
@@ -44,7 +44,7 @@ class LauncherTest {
     private static final long TIMEOUT = 2000;
 
     @Test
-    void testNotification() {
+    void notification() {
```

Comment:
Out of curiosity: what this test is testing ? I do not see any assertion nor check

Reply:
Assertion is actually done in `LauncherTest.AssertingEndpoint.notify`. In this case, `serverLauncher.getRemoteProxy().logMessage(p);` triggers jsonRPC to the client endpoint so the assertion is done on the client side. The test was not made by me, though :)

Reply:
👍